examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,82 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the regularization II
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter.
+
+In this example the dependent variable Y is set as a function
+of the input features: Y = Xa + bias. The coefficient vector a is
+randomly sampled from a normal distribution, as well as the bias
+term.
+
+As alpha tends toward zero the coefficients found by Ridge
+regression stabilize towards the randomly sampled vector a.
+For big alpha (strong regularisation) the coefficients
+are smaller (eventually converging at 0) leading to a
+simpler and biased solution.
+These dependencies can be observed on the left plot.
+
+The right plot shows the mean squared error between the
```

Comment:
@MechCoder with noise added like this:
```
features, Y, b = make_regression(n_samples=10, n_features=10, coef=True, random_state=1, bias=3.5)
features = features + np.random.randn(10, 10)
```

the curve looks different each time I sample the values (of course:), but there is no general rule. It reaches it's minimum at around alpha = 0.

I'd leave it as it is.